docs/UserGuide.adoc

```diff
@@ -373,41 +373,74 @@ Example: +
 [[listreceipt]]
 ==== Listing the current receipt: `listreceipt`
 
-Display the current receipt which shows all the items and its
-respective price and person(s) involved in splitting the cost of that item
+In this section, you will learn more about the `listreceipt` command, <<howtouselistreceiptcommand,how to use it>>
+and the <<listreceiptexpectedoutcome,expected outcome>> after using the
+`listreceipt` command.
+
+*Summary of List Receipt Command:* `listreceipt` *shows you all the entries in your current receipt*.
+
+[[howtouselistreceiptcommand]]
+*How to use the List Receipt Command:*
 
 Format: +
 `listreceipt` +
 
-Example: +
+Valid Example: +
 `listreceipt`
 
+[[listreceiptexpectedoutcome]]
+*Expected outcome:*
+
+image::expenseSplitterListAmountCommandSuccessPhoto.png[width="890", height="200"]
+
+*Additional notes and tips* +
+
+[NOTE]
+====
+* Each entry in the receipt has the item's name, price and person(s) who is splitting that item with you thus you do
+not have to remember who you split this item with!
+* When you enter `clearrreceipt` it deletes the old receipt's data. Use it with caution!
```

Comment:
Maybe instead of "Use it with caution", we can either add on, or change it to, "Only use this command when you are completely sure that you want to clear the receipt" or something equivalent. Basically a warning.